src/index.ts

```diff
@@ -197,6 +197,7 @@ export { sendMoneybackApplication } from "./services/car/moneybackApplication"
 export {
   sendBuyNowApplication,
   fetchBuyNowConfiguration,
+  markBuyNowApplicationAsPayed,
```

Comment:
i think the correct tense is "paid"
https://brians.wsu.edu/2016/05/30/payed-paid/